packages/mobile/src/identity/contactMapping.ts

```diff
@@ -124,11 +136,26 @@ export function* fetchPhoneAddresses({ e164Number }: FetchPhoneAddressesAction)
       addresses.map((a) => (addressToE164NumberUpdates[a] = e164Number))
     }
 
+    const userAddress = yield select(userAddressSelector)
```

Comment:
See other comment, please replace all `userAddressSelector` uses with the existing `currentAccountSelector`